core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1276,6 +1276,53 @@ paths:
             'application/json':
               schema:
                 $ref: '#/components/schemas/ErrorReport'
+    patch:
+      tags:
+        - submissions
+      summary: Update user comment for a submission
+      description: Update user comment for a submission
```

Comment:
I would expect the 1000 character limit to be mentioned here, since this is the API endpoint for updating the comment.

Reply:
Never mind, I see that I get the more complete description when I click on Schema.